### PR TITLE
Chartering: Update of horizontal review text and other intro matter

### PR DESCRIPTION
**Pull request by samuelweiler**
Updated all intro matter and the horizontal text.  Removed quoted bits re: patent policy, since they're in the template.

Other than markup cleanup, did not touch the w3m and ac sections.

### CODE REVIEW

**Comment by samuelweiler** [comment]
@r12a, FYI.

**Comment by r12a** [comment]
Looking good.  

It would be useful to have (a Gantt chart?) a way to see what parts of the process can overlap others, and what the dependencies are for starting/finishing a particular part of the process.

> Team Contacts(s), who will forward the requests to the Project and Strategy Leads.

How is that to be done?

typos: 
int he
drafts. use
addtional

**Comment by samuelweiler** [comment]
> > Team Contacts(s), who will forward the requests to the Project and Strategy Leads.
> 
> How is that to be done?

I'm trying to remember why I used that language, and I'm not seeing that it was there before nor in https://github.com/w3c/strategy/blob/main/4.Chartering.md. 

One open todo item (IMHO) is to merge https://github.com/w3c/strategy/blob/main/4.Chartering.md and this....  or at least have https://github.com/w3c/strategy/blob/main/4.Chartering.md point here.

**Review by wseltzer — APPROVED** [review]

File: process/charter.html [hunk 1]
```diff
@@ -82,262 +82,77 @@ <h4>Tools and templates</h4>
...
+href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">W3C Process</a> requires Team notify the Advisory Committee when a
 charter for a <strong>new</strong> Working Group or Interest Group is
 in development. The exact timing of the advance notice may vary from
 charter to charter. In practice, if the advance notice would precede
 the formal call for review by only a short delay, we skip the advance
-notice; see the <a href="https://www.w3.org/new-doc-from-template?location=%2FTeam%2F&amp;template=%2Fafs%2Fw3.org%2Fpub%2FWWW%2FTeam%2FTemplates%2Fadv-charter.html&amp;submit=Continue...">Advance Notice for Charter template</a></p>
+notice.</p>
 
-<p>The Team should consult <a href="/2005/09/adv-notice.html">How to
+<p>Consult <a href="/2005/09/adv-notice.html">How to
```
Comment (plehegar) [inline process/charter.html:125]: /2005/09/adv-notice.html is a team-only document, while /Guide is public, with the audience for each page varies. So, unless we intent to change the visibility of adv-notice, "The Team should consult" remains clearer imho.

File: process/charter.html [hunk 2]
```diff
@@ -82,262 +82,77 @@ <h4>Tools and templates</h4>
...
-and
-W3C Process Document for the
-<a href="https://www.w3.org/Consortium/Process/#WGCharter">list of charter requirements</a>.
-The Team Contact may wish to share drafts of the provisional
-charter with a candidate Chair (if one exists), a relevant interest
-group or interested community of W3C members prior to Member
-review.</p>
-
-<p class=timeline><em>Timeline:</em> due to the discussion needing to happen with the community and the need
-  for horizontal review, this takes 1.5 to 4 months for existing groups.</p>
```
Comment (plehegar) [inline process/charter.html:167]: This time estimates is useful for team contacts of far in advance to prepare their materials. It was added following a request from W3M so I'd like to keep in some form.

File: process/charter.html (hunk 1 above)
Comment (samuelweiler) [reply process/charter.html]: makes sense to me.

File: process/charter.html (hunk 2 above)
Comment (samuelweiler) [reply process/charter.html]: added this back in

**Comment by samuelweiler** [comment]
@plehegar I finally resolved your comments.  Shall we merge this?

**Comment by svgeesus** [comment]
@samuelweiler looks like you need to rebase for this to be merged

**Pull request merged**